src/display/api.js

```diff
@@ -2447,6 +2473,16 @@ class WorkerTransport {
     return this.messageHandler.sendWithPromise("GetOutline", null);
   }
 
+  getOptionalContentConfig() {
+    return this.messageHandler
+      .sendWithPromise("GetOptionalContentConfig", null)
+      .then(results => {
```

Comment:
Would it make (any) sense to cache this on the `WorkerTransport`-instance, to avoid having to wait for the worker-thread to respond for rendering to be able to start?

E.g. adding `this._optionalContentConfigPromise = null` in the `WorkerTransport` constructor and changing the code here to something like:
```javascript
if (!this._optionalContentConfigPromise) {
  this._optionalContentConfigPromise =
    this.messageHandler.sendWithPromise("GetOptionalContentConfig", null);
}
this._optionalContentConfigPromise.then(results => {
  ...
```